mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -105,6 +106,56 @@ private[feature] trait Word2VecBase extends Params
   /** @group getParam */
   def getMaxSentenceLength: Int = $(maxSentenceLength)
 
+  /**
+   * Number of negative samples to use with CBOW based estimation.
+   * This parameter is ignored for SkipGram-Hierachical Softmax based estimation.
```

Comment:
Rather than exclusion, I would recommend to use precise description here and in other places.
The parameter is only used for continuous bag-of-words with negative sampling algorithm, i.e. solver = "cbow-ns".